src/nwb_conversion_tools/datainterfaces/ophys/scanimage/scanimageimaginginterface.py

```diff
@@ -49,12 +50,17 @@ def __init__(
         super().__init__(file_path=file_path, sampling_frequency=sampling_frequency)
 
     def get_metadata(self):
+        device_number = 0  # Imaging plane metadata is a list with metadata for each plane
+
+        metadata = super().get_metadata()
 
-        new_metadata = dict(Ophys=dict(TwoPhotonSeries=dict(description=json.dumps(self.image_metadata))))
+        extracted_session_start_time = dateparse(self.image_metadata["state.internal.triggerTimeString"])
+        metadata["NWBFile"] = dict(session_start_time=extracted_session_start_time)
 
-        if "state.internal.triggerTimeString" in self.image_metadata:
-            new_metadata["NWBFile"] = dict(
-                session_start_time=dateparse(self.image_metadata["state.internal.triggerTimeString"])
-            )
```

Comment:
why did you change this?

Reply:
Now if this field is not present the entire conversion fails

Reply:
When I modified the metadata to wrap the dicts into list I was getting problems matching the data to the expected results in the test (as everything was string) so I re-wrote everything to make the comparison specific to each of the levels. The failure mode that you pointed out is something that did not foresee. 

This could be fixed either by testing the present of the extracted_session_time or by writing everything to a temporary metadata dict and then using `deep_dict_update` as you had it before. I prefer the former, do you have a preference?

Reply:
Btw, while lookin at the extracted json I saw the following:

```
    "state.internal.startupTimeString": "\'10/9/2017 14:38:30.957\'",
    "state.internal.triggerTimeString": "\'10/9/2017 16:57:07.967\'",
```
There are two time strings over there, how did you chose between those two?

Reply:
I would guess the data collection starts at the trigger time

Reply:
I don't really have a preference for how we add the data to the dict as long as we are robust to missing metadata

Reply:
Good. Now I added that. I also added a new assertion in case the sampling frequency was not found.